src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheckTest.java

```diff
@@ -72,6 +72,22 @@ public void testMissingSwitchDefaultSwitchExpressions() throws Exception {
             expected);
     }
 
+    @Test
+    public void testNullCaseLabel() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWithInlineConfigParser(
+                getNonCompilablePath("InputMissingSwitchDefaultCheckNullCaseLabel.java"),
+            expected);
+    }
+
+    @Test
+    public void testNullCaseLabelTwo() throws Exception {
```

Comment:
See https://github.com/checkstyle/checkstyle/pull/14070#discussion_r1407657247 again.

Reply:
Removed and used only one @nrmancuso